pinc/bad_bytes.inc

```diff
@@ -786,26 +786,21 @@ define_bad_byte_sequences();
 
 // XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 
-function find_bad_byte_sequences_in_text($text, $projectid)
+define("BOM", "\u{feff}");
+
+function find_bad_byte_sequences_in_text(&$text)
```

Comment:
I really don't like pass-by-reference in PHP because it's unexpected and confusing since almost everything else is pass-by-value. Please add a comment that this will modify the value of `$text`, both here and where it is called.

Reply:
Done this.